### PR TITLE
Redesign/courses

### PR DESCRIPTION
**Pull request by MichaelBooi**
- Container created
- Card creation
- Image added
- Button links added
- Slider added
- Slider arrows added (not the style yet like requested)
- Background color gray
- Title added
- Space between cards
- Arrow placement
- Button text color is white instead of yellow now

Sorry for the long list, I will commit changes faster from now on!
This is my current progress so far.

screenshot from 2016-10-26 17-08-57

### CODE REVIEW

File: app/assets/stylesheets/config/_layout.sass [hunk 1]
```diff
@@ -1,22 +1,37 @@
...
+
+.card .card-image img
+  height: 150px
+
+.card .card-action
+  border-top: 0px
+  padding: 0px
+
+
+
```
Comment (miriamtocino) [inline app/assets/stylesheets/config/_layout.sass]: Same comment as for @tanjahennis . Every file should only contain one extra line at the end. She already set it up for RubyMine so ask her how to do this 😄

File: app/assets/javascripts/frontend/courses.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    if (n < 1) {slideIndex = slides.length}
+    for (i = 0; i < slides.length; i++) {
+        slides[i].style.display = "none";
+    }
+    for (i = 0; i < dots.length; i++) {
+        dots[i].className = dots[i].className.replace(" active", "");
+    }
+    slides[slideIndex-1].style.display = "block";
+    dots[slideIndex-1].className += " active";
+}
```
Comment (miriamtocino) [inline app/assets/javascripts/frontend/courses.js]: RubyMine will take care of adding an extra line when you set it up!

Comment (miriamtocino) [inline app/assets/javascripts/frontend/courses.js]: Also you should set up your tabs to be 2 spaces wide. I think you have it set to 4...

File: config/environments/development.rb [hunk 3]
```diff
@@ -14,8 +14,8 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
```
Comment (miriamtocino) [inline config/environments/development.rb]: You should revert these changes, since as far as I know, they were not related to this project :wink:

Comment (MichaelBooi) [inline config/environments/development.rb]: Done

File: app/assets/stylesheets/config/_layout.sass (hunk 1 above)
Comment (MichaelBooi) [inline app/assets/stylesheets/config/_layout.sass]: We both don't seem to get it to work, I tried Googling it but no results. I'll ask michal tomorrow.

File: app/assets/javascripts/frontend/courses.js (hunk 2 above)
Comment (MichaelBooi) [inline app/assets/javascripts/frontend/courses.js]: I wasted too much time on trying to get the line thing and it's already 19:00 so I'll do this tomorrow :sleepy:

**Comment by MichaelBooi** [comment]
Added a more fitting picture :)

screenshot from 2016-10-26 19-11-41

File: app/assets/stylesheets/config/_layout.sass (hunk 1 above)
Comment (MichaelBooi) [inline app/assets/stylesheets/config/_layout.sass]: I think it actually works now :open_mouth:

File: app/assets/javascripts/frontend/courses.js (hunk 2 above)
Comment (MichaelBooi) [inline app/assets/javascripts/frontend/courses.js]: I've set the tab spaces to be 2 wide as default now!

**Comment by miriamtocino** [comment]
LGTM @MichaelBooi !
🚀 🚀 🚀
